docs/_layouts/release.html

```diff
@@ -108,7 +108,7 @@ <h2>Channel mappings</h2>
         {% assign rules = mapping.rules %}
         <h3 class="value-title value-title--box mapping-title" tabindex="0">
           {% if output_title == "shopify" %}
-            {{ output_title }} (Previous Versions)
+            {{ output_title }} (Legacy)
```

Comment:
Fine for now, but what happens when we have a mapping from `2024-07`?

Reply:
From what I gathered each _release_ will be mapped to this legacy version (ver 2022-02).

Reply:
Mappings are to the previous version (mapping back to "legacy" would be computed by following mappings release by releaes)